fmriprep/workflows/anatomical.py

```diff
@@ -74,43 +74,44 @@ def t1w_preprocessing(settings, name='t1w_preprocessing'):
             testing=settings.get('debug', False),
             template='mni_icbm152_nlin_asym_09c'
         ),
-        name='T1_2_MNI_Registration'
+        name='t1_2_mni'
     )
     # should not be necesssary but does not hurt - make sure the multiproc
     # scheduler knows the resource limits
     t1_2_mni.interface.num_threads = settings['ants_nthreads']
 
     # 6. FreeSurfer reconstruction
     if settings['freesurfer']:
-        surface_recon = surface_reconstruction(settings=settings)
+        surface_recon = surface_reconstruction(name='surface_recon',
```

Comment:
Yeah I was wondering about this as well - what's the best way to do this. My reasoning was that:
- you are interested in the instance of a particular workflow not the name of the function (which is helpful when distinguishing between multiple instances of the same workflow)
- this would be consistent with the Node naming scheme

However, in FMRIPREP we almost never reuse functions for creating workflows. Therefore in all cases "instance == function" and naming workflow after the function actually makes it faster to find relevant code.